benchmarl/conf/experiment/base_experiment.yaml

```diff
@@ -26,6 +26,8 @@ gamma: 0.99
 lr: 0.00005
 # The epsilon parameter of the adam optimizer
 adam_eps: 0.000001
+# Extra kwargs for the adam optimizer
+adam_extra_kwargs: {}
```

Comment:
In preparation for the change you mentioned (aka allow any optimiser) we could call this 
`optimizer_extra_kwargs`